client/logic/game/game.gd

```diff
@@ -1241,13 +1239,21 @@ func _create_and_set_attack_area(
 				return
 	var terrains: Array[Terrain] = unit.get_possible_terrains_to_attack_from_terrain(target_terrain)
 	for i: Terrain in terrains:
```

Comment:
I'm the big fan of one letter variables, but what about having some one letters more descriptives? or maybe better a more long variable? Now the loop grown, I think will be better